articles/aks/open-ai-quickstart.md

```diff
@@ -105,6 +105,9 @@ To manage a Kubernetes cluster, use the Kubernetes command-line client, [kubectl
     aks-nodepool1-31469198-vmss000001   Ready    agent   3h29m   v1.25.6
     aks-nodepool1-31469198-vmss000002   Ready    agent   3h29m   v1.25.6
     ```
+   
+> [!NOTE]
+> For private clusters, the nodes might be unreachable if you try to connect to them through the public IP address. In order to fix this, you need to create an end-point within the same VNET as the cluster to connect from. Follow the instructions to [Create a private AKS cluster] and connect to it.
```

Comment:
replace end-point with endpoint. Suggest rewording last sentence to - Follow the instructions to [Create a private AKS cluster] and then connect to it.